### ISSUE TITLE
How to test 

### ISSUE THREAD

**Issue by jingyuerY** [LIVIAETS/boxes_tightness_prior#4]
How to test the effect of the trained network on the test set？Thank you very much！

**Comment by HKervadec** [comment]
You simply need to create a dataloader like for the validation set, and use the same inference loop (`do_epoch` in validation mode), [like there](https://github.com/LIVIAETS/boxes_tightness_prior/blob/master/main.py#L348).
